### DISCUSSION TITLE
Memorised vs seen progress - hints?

### DISCUSSION THREAD

**Discussion by YannCebron** [didoesdigital/typey-type#95] [Q&A]
I noticed that my _memorised_ count was only at about 10% of total seen words. Would it make sense to push users in some way to keep this at a different (higher) level? It seems to me, I went "too fast" and should concentrate on increasing my count of memorised words first, before adding more new words.

Maybe some % indication with traffic light indicator or similar..?

**Comment by didoesdigital** [comment]
Great idea @YannCebron ! I wonder what would make for a good memorised target? 🤔

**Comment by YannCebron** [comment]
not sure this could be generalized.. maybe a user setting and default of 20..30%?

**Comment by didoesdigital** [comment]
Hmm, maybe we could add something around here on [the progress page](https://didoesdigital.com/typey-type/progress)?
<img width="434" alt="image" src="https://user-images.githubusercontent.com/2476974/202844934-2cf3930f-7d97-4512-b811-326182938319.png">

<img width="401" alt="image" src="https://user-images.githubusercontent.com/2476974/202844917-665602aa-81dd-4056-8101-1ca05155f5da.png">

**Comment by YannCebron** [comment]
yes, that makes perfectly sense

**Comment by didoesdigital** [comment]
@YannCebron are you happy with where your accuracy is at for the majority of words you need to type? I was pondering good targets for memorised percentages and where/when we might prompt folk to focus on revision vs new words. I was wondering how accuracy with words you care about compares to % memorised as a measure of progress and possible feedback.

**Comment by YannCebron** [comment]
I'd argue those might be two different metrics. For me, I struggle sometimes with "new words" which I simply do not remember at all yet - having typed them maybe only once or twice - so my accuracy for those is 0%. Maybe time for a poll in discord? :-)

**Reply by didoesdigital** [reply]
Go for it!

**Comment by YannCebron** [comment]
https://discord.com/channels/136953735426473984/136953809409802240/1043467493433409588
